pkg/agent/datamodel/sig_config.go

```diff
@@ -260,191 +266,236 @@ const (
 	EdgeZoneSIGImageVersion string = "2023.01.09"
 )
 
-// SIG config Template
+var LinuxSIGImageVersion string
+
 var (
-	SIGUbuntu1604ImageConfigTemplate = SigImageConfigTemplate{
-		ResourceGroup: AKSUbuntuResourceGroup,
-		Gallery:       AKSUbuntuGalleryName,
-		Definition:    "1604",
-		Version:       "2021.11.06",
-	}
+	SIGUbuntuContainerd1804Gen2ImageConfigTemplate,
```

Comment:
suggestion: create a function that returns the populated versions, not an init func that populates the vars

Reply:
I'd created the previous initialization helper function, because it was needed when using an init function (since the original package variables are evaluated before the init function).
With the removal of the init function, the helper function is also not needed.